src/utils/network.cpp

```diff
@@ -22,6 +25,9 @@ make_http_request(HTTPMethod method,
     httplib::Client client(base_url);
     client.set_follow_location(follow_location);
 
+    network_log->log(LogLevel::INFO, LogOutput::FILE, "Making HTTP request to: " + base_url + route_url);
```

Comment:
Utils should not log. It's the caller's job to check the returned result and do logging if needed. Utils don't care about logging.